tensorflow_io/core/kernels/avro/parse_avro_kernels.cc

```diff
@@ -204,8 +207,13 @@ Status ParseAvro(const AvroParserConfig& config,
         minibatch_bytes = 0;
       }
     }
-    // 'special logic'
-    const size_t min_minibatches = std::min<size_t>(8, serialized.size());
+    if (avro_num_minibatches_) {
+      VLOG(5) << "Overriding num_minibatches with " << avro_num_minibatches_;
+      result = avro_num_minibatches_;
+    }
+    // This is to ensure users can control the num minibatches all the way down
+    // to size of 1(no parallelism).
+    const size_t min_minibatches = std::min<size_t>(1, serialized.size());
```

Comment:
can you also comment that avro_num_minibatches_ is [1, 64] in the op interface?

Reply:
Updated